harpy/snp.py

```
@@ -109,30 +110,29 @@ def mpileup(inputs, output_dir, regions, genome, threads, populations, ploidy, e
-
+    if populations:
+        validate_popfile(populations)
+        # check that samplenames and populations line up
+        validate_popsamples(bamlist, populations, quiet)
+    configs = {
+        "workflow" : "snp mpileup",
+        "snakemake_log" : sm_log,
+        "output_directory" : output_dir,
+        "ploidy" : ploidy,
+        "regiontype" : regtype,
+        **({'windowsize': regions} if regtype is "windows" else {}),
```

Ruff

Use `==` to compare constant literals

Replace `is` with `==`

(F632)